src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -199,6 +199,19 @@ public BillingProfileModel getProfileByIdNoCheck(UUID id) {
     return profileDao.getBillingProfileById(id);
   }
 
+  public boolean isTdrBillingProfile(UUID defaultProfileId) {
+    // Locate billing profile in TDR or Rawls
+    // No auth check: Just a check if there is an entry in our db for this billing profile
+    boolean isTdrBillingProfile;
+    try {
+      getProfileByIdNoCheck(defaultProfileId);
+      isTdrBillingProfile = true;
```

Comment:
Up to you, but now that this is a function you can get `return` here and in the `catch` and remove `isTdrBillingProfile` from the function.